daemon/web/pages/settings/Settings.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import ShutdownButton from '../../components/ShutdownButton/ShutdownButton';
+import IconHeader from '../../components/IconHeader/IconHeader';
+
+
+import {
+  Table,
+  TableCell,
+  TableRow,
+  TableHeader,
+  TableBody,
+} from '../../components/Table/Table';
+
+class SettingsWrapper extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+
+      <div>
+        <IconHeader type="settings" title="PROJECT INFORMATION" style={{ margin: '1rem' }} />
+        <Table style={{ width: '90%', margin: '1rem' }}>
+          <TableHeader>
+            <TableRow>
+              <TableCell style={{ fontWeight: '900', fontSize: '14px' }}>Project-name</TableCell>
+              <TableCell />
+              <TableCell />
+            </TableRow>
+          </TableHeader>
+
+          <TableBody>
+            <TableRow>
```

Comment:
Sure you can list out the divs like this, but think about what the API will returns to the web app:

https://github.com/ubclaunchpad/inertia/blob/master/common/types.go#L4

```go
type DeploymentStatus struct {
	InertiaVersion       string   `json:"version"`
	Branch               string   `json:"branch"`
	CommitHash           string   `json:"commit_hash"`
	CommitMessage        string   `json:"commit_message"`
	BuildType            string   `json:"build_type"`
	Containers           []string `json:"containers"`
	BuildContainerActive bool     `json:"build_active"`
}
```

maybe there's a cleaner way to generate this table?

Reply:
right the html is on the redundant side as i was trying to get a more consistent look. I'll get back to this